### PR TITLE
XSIAM_PB_Event_Log_Was_Cleared

### PR DESCRIPTION
**Pull request by OmriItzhak**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://jira-dc.paloaltonetworks.com/browse/CIAC-10784

## Description
A new core playbook and trigger for XSIAM designed to handle the alerts:
- Windows Event Log was cleared using wevtutil.exe
- Security Event Log was cleared using wevtutil.exe
- A Sensitive Windows Event Log was cleared using wevtutil.exe
- Windows event logs were cleared with PowerShell
- Suspicious clear or delete security provider event logs with PowerShell
- Suspicious clear or delete default providers event logs with PowerShell
- Windows event logs cleared using wmic.exe

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/Core/Playbooks/playbook-Event_Log_Was_Cleared.yml [hunk 1]
```diff
@@ -0,0 +1,569 @@
+id: Event Log Was Cleared
+version: -1
+name: Event Log Was Cleared
+description: "This playbook is designed to handle the following alerts:  \n- Windows Event Log was cleared using wevtutil.exe\n- Security Event Log was cleared using wevtutil.exe\n- A Sensitive Windows Event Log was cleared using wevtutil.exe\n- Windows event logs were cleared with PowerShell\n- Suspicious clear or delete security provider event logs with PowerShell\n- Suspicious clear or delete default providers event logs with PowerShell\n- Windows event logs cleared using wmic.exe\n\nThe playbook executes the following stages:\n\nInvestigation:\nCheck the following parameters to determine if remediation actions are needed:\n- XDR alerts related to the hostname by MITRE tactics indicating malicious activity.\n- Whether the process is signed.\n- The prevalence of the process.\n\nRemediation:\nHandles malicious alerts by terminating the relevant processes.\nHandles non-malicious alerts identified during the investigation."
```
Comment (ShirleyDenkberg) [inline Packs/Core/Playbooks/playbook-Event_Log_Was_Cleared.yml:4]:
```suggestion
description: "This playbook is designed to handle the following alerts:  \n- Windows Event Log was cleared using wevtutil.exe\n- Security Event Log was cleared using wevtutil.exe\n- A Sensitive Windows Event Log was cleared using wevtutil.exe\n- Windows event logs were cleared with PowerShell\n- Suspicious clear or delete security provider event logs with PowerShell\n- Suspicious clear or delete default providers event logs with PowerShell\n- Windows event logs cleared using wmic.exe\n\nThe playbook executes the following stages:\n\nInvestigation:\nCheck the following parameters to determine if remediation actions are needed:\n- Cortex XDR alerts related to the hostname by MITRE tactics indicating malicious activity.\n- Whether the process is signed.\n- The prevalence of the process.\n\nRemediation:\nHandles malicious alerts by terminating the relevant processes.\nHandles non-malicious alerts identified during the investigation."
```

File: Packs/Core/Playbooks/playbook-Event_Log_Was_Cleared.yml [hunk 2]
```diff
@@ -0,0 +1,569 @@
...
+    isautoswitchedtoquietmode: false
+  "76":
+    id: "76"
+    taskid: 7c9f23f6-7986-4c42-835f-f31c037a9fde
+    type: regular
+    task:
+      id: 7c9f23f6-7986-4c42-835f-f31c037a9fde
+      version: -1
+      name: Terminate the causality process
+      description: Terminate a process tree by its causality ID. Available only for XSIAM 2.4.
```
Comment (ShirleyDenkberg) [inline Packs/Core/Playbooks/playbook-Event_Log_Was_Cleared.yml:421]:
```suggestion
      description: Terminate a process tree by its causality ID. Available only for Cortex XSIAM 2.4.
```

File: Packs/Core/Playbooks/playbook-Event_Log_Was_Cleared.yml [hunk 3]
```diff
@@ -0,0 +1,569 @@
...
+  }
+inputs: []
+inputSections:
+- inputs: []
+  name: General (Inputs group)
+  description: Generic group for inputs
+outputSections:
+- outputs: []
+  name: General (Outputs group)
+  description: Generic group for outputs
```
Comment (ShirleyDenkberg) [inline Packs/Core/Playbooks/playbook-Event_Log_Was_Cleared.yml:565]:
```suggestion
  description: Generic group for inputs.
outputSections:
- outputs: []
  name: General (Outputs group)
  description: Generic group for outputs.
```

File: Packs/Core/Playbooks/playbook-Event_Log_Was_Cleared_README.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+- Windows event logs were cleared with PowerShell
+- Suspicious clear or delete security provider event logs with PowerShell
+- Suspicious clear or delete default providers event logs with PowerShell
+- Windows event logs cleared using wmic.exe
+
+The playbook executes the following stages:
+
+Investigation:
+Check the following parameters to determine if remediation actions are needed:
+- XDR alerts related to the hostname by MITRE tactics indicating malicious activity.
```
Comment (ShirleyDenkberg) [inline Packs/Core/Playbooks/playbook-Event_Log_Was_Cleared_README.md:14]:
```suggestion
- Cortex XDR alerts related to the hostname by MITRE tactics indicating malicious activity.
```

File: Packs/Core/Playbooks/playbook-Event_Log_Was_Cleared_README.md [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+
+This playbook uses the following sub-playbooks, integrations, and scripts.
+
+### Sub-playbooks
+
+This playbook does not use any sub-playbooks.
+
+### Integrations
+
+* CortexCoreIR
```
Comment (ShirleyDenkberg) [inline Packs/Core/Playbooks/playbook-Event_Log_Was_Cleared_README.md:32]:
```suggestion
CortexCoreIR
```

File: Packs/Core/Playbooks/playbook-Event_Log_Was_Cleared_README.md [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+
+### Scripts
+
+* SearchAlertsV2
```
Comment (ShirleyDenkberg) [inline Packs/Core/Playbooks/playbook-Event_Log_Was_Cleared_README.md:36]:
```suggestion
SearchAlertsV2
```

File: Packs/Core/ReleaseNotes/3_0_71.md [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+##### New: Event Log Was Cleared
+
+New: This playbook is designed to handle the following alerts:  
+Windows Event Log was cleared using wevtutil.exe
+Security Event Log was cleared using wevtutil.exe
+A Sensitive Windows Event Log was cleared using wevtutil.exe
+Windows event logs were cleared with PowerShell
+Suspicious clear or delete security provider event logs with PowerShell
+Suspicious clear or delete default providers event logs with PowerShell
+Windows event logs cleared using wmic.exe
```
Comment (ShirleyDenkberg) [inline Packs/Core/ReleaseNotes/3_0_71.md:13]:
```suggestion
- Windows Event Log was cleared using wevtutil.exe
- Security Event Log was cleared using wevtutil.exe
- A Sensitive Windows Event Log was cleared using wevtutil.exe
- Windows event logs were cleared with PowerShell
- Suspicious clear or delete security provider event logs with PowerShell
- Suspicious clear or delete default providers event logs with PowerShell
- Windows event logs cleared using wmic.exe
```

File: Packs/Core/ReleaseNotes/3_0_71.md [hunk 8]
```diff
@@ -0,0 +1,32 @@
...
+
+The playbook executes the following stages:
+
+Investigation:
+Check the following parameters to determine if remediation actions are needed:
+XDR alerts related to the hostname by MITRE tactics indicating malicious activity.
+Whether the process is signed.
+The prevalence of the process.
```
Comment (ShirleyDenkberg) [inline Packs/Core/ReleaseNotes/3_0_71.md:21]:
```suggestion
- Cortex XDR alerts related to the hostname by MITRE tactics indicating malicious activity.
- Whether the process is signed.
- The prevalence of the process.
```

File: Packs/Core/ReleaseNotes/3_0_71.md [hunk 9]
```diff
@@ -0,0 +1,32 @@
...
+
+Remediation:
+Handles malicious alerts by terminating the relevant processes.
+Handles non-malicious alerts identified during the investigation.
```
Comment (ShirleyDenkberg) [inline Packs/Core/ReleaseNotes/3_0_71.md:25]:
```suggestion
- Handles malicious alerts by terminating the relevant processes.
- Handles non-malicious alerts identified during the investigation.
```

**Comment by ShirleyDenkberg** [comment]
@OmriItzhak Doc review completed.

File: Packs/Core/Playbooks/playbook-Event_Log_Was_Cleared_README.md [hunk 10]
```diff
@@ -0,0 +1,58 @@
...
+- Suspicious clear or delete security provider event logs with PowerShell
+- Suspicious clear or delete default providers event logs with PowerShell
+- Windows event logs cleared using wmic.exe
+
+The playbook executes the following stages:
+
+Investigation:
+Check the following parameters to determine if remediation actions are needed:
+- XDR alerts related to the hostname by MITRE tactics indicating malicious activity.
+- Whether the process is signed.
```
Comment (ShirleyDenkberg) [inline Packs/Core/Playbooks/playbook-Event_Log_Was_Cleared_README.md:15]: Should this be signed or unsigned?

File: Packs/Core/ReleaseNotes/3_0_72.md [hunk 11]
```diff
@@ -0,0 +1,32 @@
...
+Suspicious clear or delete default providers event logs with PowerShell
+Windows event logs cleared using wmic.exe
+
+The playbook executes the following stages:
+
+Investigation:
+Check the following parameters to determine if remediation actions are needed:
+XDR alerts related to the hostname by MITRE tactics indicating malicious activity.
+Whether the process is signed.
+The prevalence of the process.
```
Comment (ssokolovich) [inline Packs/Core/ReleaseNotes/3_0_72.md:21]: of which process?

File: Packs/Core/ReleaseNotes/3_0_72.md [hunk 12]
```diff
@@ -0,0 +1,32 @@
...
+Suspicious clear or delete security provider event logs with PowerShell
+Suspicious clear or delete default providers event logs with PowerShell
+Windows event logs cleared using wmic.exe
+
+The playbook executes the following stages:
+
+Investigation:
+Check the following parameters to determine if remediation actions are needed:
+XDR alerts related to the hostname by MITRE tactics indicating malicious activity.
+Whether the process is signed.
```
Comment (ssokolovich) [inline Packs/Core/ReleaseNotes/3_0_72.md:20]:
```suggestion
Whether the action generated process is signed or not.
```

File: Packs/Core/Playbooks/playbook-Event_Log_Was_Cleared.yml [hunk 13]
```diff
@@ -0,0 +1,569 @@
...
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "73":
+    id: "73"
+    taskid: 11e37dbd-4664-442a-8b48-737b5e95ad75
+    type: regular
+    task:
+      id: 11e37dbd-4664-442a-8b48-737b5e95ad75
+      version: -1
+      name: 'Get prevalence for the processes in the causality '
```
Comment (ssokolovich) [inline Packs/Core/Playbooks/playbook-Event_Log_Was_Cleared.yml:369]:
```suggestion
      name: Get prevalence for the processes in the causality
```

File: Packs/Core/Playbooks/playbook-Event_Log_Was_Cleared_README.md (hunk 10 above)
Comment (OmriItzhak) [reply Packs/Core/Playbooks/playbook-Event_Log_Was_Cleared_README.md:15]: unsigned

File: Packs/Core/ReleaseNotes/3_0_71.md (hunk 7 above)
Comment (OmriItzhak) [reply Packs/Core/ReleaseNotes/3_0_71.md]: Done

File: Packs/Core/ReleaseNotes/3_0_71.md (hunk 9 above)
Comment (OmriItzhak) [reply Packs/Core/ReleaseNotes/3_0_71.md]: Done

File: Packs/Core/ReleaseNotes/3_0_72.md (hunk 11 above)
Comment (OmriItzhak) [reply Packs/Core/ReleaseNotes/3_0_72.md]: Done. 
changed to
"The prevalence of the OSParent process."

File: Packs/Core/ReleaseNotes/3_0_72.md (hunk 12 above)
Comment (OmriItzhak) [reply Packs/Core/ReleaseNotes/3_0_72.md]: Done. changed to "Whether the CGO or the OSParent process is unsigned."

**Review by umishkin — APPROVED** [review]

**Review by ssokolovich — APPROVED** [review]

**Pull request merged**